keps/sig-release/0000-anago-to-krel-migration/kep.yaml

```diff
@@ -7,4 +7,5 @@ reviewers:
 approvers:
   - "@justaugustus"
 creation-date: 2020-09-22
-status: implementable
+latest-milestone: v1.20
```

Comment:
The KEP has still number `0000`, which one do we want to assign them?

Reply:
The next incremental would be 1735, do the numbers have any special meaning?

Reply:
Yes, I thought they were related to the issue number, but unfortunately we have no corresponding issue.